new/vcloud-manager-blueprint-inputs.yaml

```diff
@@ -0,0 +1,19 @@
+management_network_name: some_managements_network
```

Comment:
Can you please format this file as it is formatted in the openstack/simple manager blueprints?